### ISSUE TITLE
Can't update extensions (ungoogled-chromium 110.0.5481.84)

### ISSUE THREAD

**Issue by jniggemann** [NeverDecaf/chromium-web-store#115]
After the last update, I can no longer update extensions. I get a notification that there's an update for NoScript, but no matter which tab  I have open when clicking on chromium web store, I get the message "No apps, extensions or user scripts can be installed from this website".

![image](https://user-images.githubusercontent.com/1318059/217810755-7a8a9fb6-2a5c-46eb-bf1d-fb6ffc1dffb0.png)

**Comment by NeverDecaf** [comment]
When you say last update, do you mean from a previous version of this extension? If so, can you tell me which version was working for you in the past?

This would be my response otherwise:
If the `#extension-mime-request-handling` flag does not work for you then it is not possible to install or update extensions directly. You can check the `Always download CRX files` box in options to make manual install a little easier.

**Comment by jniggemann** [comment]
Sorry for not being clearer: After updating _chromium_, not the extension. With `Always download CRX files` NOT set, I observed that the extension file is downloaded all right, and then the message in my screenshot shows. I can then proceed to drag the the crx file to "manage extensions" (my translation, I mean the extensions overview page) and it installs all right.

**Comment by NeverDecaf** [comment]
If it worked in previous versions of chromium then this is likely an issue with your browser, and not the extension. I don't know if this helps but I am using [Version 110.0.5481.84 (Official Build, ungoogled-chromium) (64-bit)](https://github.com/macchrome/winchrome/releases) and haven't experienced any problems.

**Comment by jniggemann** [comment]
Further testing cws 1.5.3 with a new and empty profile in the same chromium version I experienced the error in ([Version 110.0.5481.104 (Official Build, ungoogled-chromium) (64-Bit)](https://github.com/macchrome/winchrome/releases/tag/v110.5481.104-M110.0.5481.104-r839-Win64), same source you use) revealed that it works normally.

This means that it's probably a profile issue (other extensions, settings, whatever).

Closing this, as I don't have time to investigate.
